tests/integration/v4_slowquery_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"gorm.io/gorm"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/slowquery"
+	"github.com/pingcap/tidb-dashboard/pkg/utils"
+	"github.com/pingcap/tidb-dashboard/util/testutil"
+)
+
+type testV4SlowQuerySuite struct {
+	suite.Suite
+	db               *testutil.TestDB
+	slowqueryColumns []string
+}
+
+func TestV4SlowQuery(t *testing.T) {
+	if !strings.Contains(os.Getenv("TIDB_VERSION"), "v4") {
+		t.Skip()
+	}
+
+	db := testutil.OpenTestDB(t)
+	suite.Run(t, &testV4SlowQuerySuite{
+		db: db,
+	})
+}
+
+func (s *testV4SlowQuerySuite) SetupSuite() {
+	s.prepareSlowQuery()
+
+	columns, err := utils.NewSysSchema().GetTableColumnNames(s.slowQuerySession(), slowquery.SlowQueryTable)
+	s.NoError(err)
+	s.slowqueryColumns = columns
+}
+
+func (s *testV4SlowQuerySuite) TearDownSuite() {
+	s.db.MustClose()
+}
+
+func (s *testV4SlowQuerySuite) prepareSlowQuery() {
+	s.db.MustExec("SET tidb_slow_log_threshold = 0")
+	var wg sync.WaitGroup
+	for i := 1; i < 5; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			s.db.MustExec(fmt.Sprintf("SELECT count(*) FROM %s", slowquery.SlowQueryTable))
```

Comment:
If you are trying to simulate slow queries, then something like `sleep(1)` could be your friend.

Reply:
Maybe `sleep(1)` will slow down the tests?

Reply:
One second sleep should be fine, as long as you don't have too many of them LOL